src/components/OperationalStatus/OperationalStatus.stories.tsx

```diff
@@ -0,0 +1,43 @@
+import { Meta, StoryFn } from "@storybook/react";
+import { OperationalStatus, OperationalStatusData } from "./OperationalStatus";
+import jsonData from "../../../public/status.json";
```

Comment:
To keep the story static, omit this line and embed the data with the story args, you can set up a Primary story and the the other story can deconstruct it's arg into theirs. Check Page.stories.tsx for an example

Reply:
Updated